manual/src/refman/expr.etex

```diff
@@ -437,7 +437,7 @@ and allow one to use this polymorphism in recursive occurrences
 (when using @"let" "rec"@). Note however that this is a normal polymorphic
 type, unifiable with any instance of itself.
 
-It is possible to define local exceptions in expressions:
+It is possible to define local exceptions in expressions since 4.04:
```

Comment:
This is a good idea;  a possible improvement would to mimic the subsubsection above:
```latex
\subsubsection{sss:expr-local-exception}{Local exceptions}
(Introduced in 4.04)
```